src/Security/Authorization/Core/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Runtime.CompilerServices;
+using Microsoft.AspNetCore.Authorization;
+
+[assembly: TypeForwardedTo(typeof(IAuthorizeData))]
```

Comment:
Where these interfaces used by anyone but us prior to 3.0? What for?

Reply:
By anyone who wanted to? They are public.